server/sonar-ce/src/test/java/org/sonar/ce/configuration/CeConfigurationRule.java

```diff
@@ -31,6 +31,8 @@
   private long queuePollingDelay = 2 * 1000L;
   private long cancelWornOutsInitialDelay = 1L;
   private long cancelWornOutsDelay = 10L;
+  private long tasksWithUnknownWorkerUUIDsInitialDelay = 1L;
```

Comment:
I'd use a different initial delay from the cancelWornOuts job, just to not shoot ourselves in the foot